frontend/ui/size.lua

```diff
@@ -77,6 +89,11 @@ local Size = {
         vertical_default = Screen:scaleBySize(2),
         vertical_large = Screen:scaleBySize(5),
     },
+    maxItemsNorm = {
+        info_list = size_max_items_norm(Screen:getHeight(), Device.display_dpi, 20),
+        mosaic_h = size_max_items_norm(Screen:getHeight(), Device.display_dpi, 75),
+        mosaic_w = size_max_items_norm(Screen:getWidth(), Device.display_dpi, 75),
```

Comment:
As it is well known, I sucks at maths and DPI.
I don't know how this differs from the size we may get with scaleBySize() et al. stuff we have in base/ffi/framebuffer.lue. Except may be by being specifically scaled/constrained to the device height **or** its width (unlike the other which takes the min of them) ?

I also don't know how much it's relevant or overengineererd as it's for specifying some max in SpinWidgets, and we could well go with some random hardcoded max.

Reply:
> and we could well go with some random hardcoded max.

This is exactly what I am trying to fix :D 
KOReader needs to be smarter about sizes.

Reply:
`scaleBySize` does something slightly custom, as it was engineered to get matching layouts on a K2/K3 vs. a DX/DXG.

This is based on the standard dpi-aware pt-to-px conversion people familiar with typesetting are probably familiar with ;).

e.g., https://github.com/NiLuJe/FBInk/blob/f7f80f019f7c32cedc0f4aa6b717708fbed2472b/fbink.c#L7241-L7244

Reply:
Thanks @NiLuJe !
Yes, this was my intention.
I'll make a diagram later to explain it better.